packages/encryption/package.json

```diff
@@ -31,11 +31,11 @@
   },
   "dependencies": {
     "@stacks/common": "^2.0.1",
-    "@types/bn.js": "^4.11.6",
+    "@types/bn.js": "^5.1.0",
     "@types/node": "^14.14.43",
     "bip39": "^3.0.2",
     "bitcoinjs-lib": "^5.2.0",
```

Comment:
IIRC it was the `bitcoinjs` family of libs (this one and some of the `bip` libs) that were including `bn.js` v4. Have you tested that these dependencies don't end up requiring `bn.js` v4 to be bundled anyway?

Reply:
Other packages are depending of v4, my idea was to do the upgrade gradually, here is the list of deps still using it (might have missed some):

- @stacks/keychain
  - @blockstack/rpc-client 0.3.0-alpha.11
    - @blockstack/stacks-transactions 0.5.1
- Used by multiple packages
  - jsontokens 3.0.0
    - asn1.js 5.4.1
- @stacks/cli 2.0.1
  - blockstack 19.3.0
- Used by multiple packages
  - elliptic 6.4.5
- bitcoin-lib 5.2.0
  - tiny-secp256k1 1.1.6

The issue can be that some of them seem unmaintained, like https://github.com/indutny/asn1.js/pull/118 where the pr was opened almost 2 years ago.